src/content/guides/migrating.md

```diff
@@ -544,7 +544,7 @@ webpack now supports template strings in expressions. This means you can start u
 
 ### Configuration Promise
 
-webpack now supports returning a `Promise` from the configuration file. This allows to do async processing in you configuration file.
+webpack now supports returning a `Promise` from the configuration file. This allows to do async processing in your configuration file.
```

Comment:
We should probably clean up that last sentence even more:

> This allows async processing in your configuration file.

Or even combine the whole thing to one:

> webpack now supports returning a `Promise` from the configuration file which allows async processing in your configuration file.